### PR TITLE
o/cincinnati-operator: test on all OCPs

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml [hunk 1]
```diff
@@ -97,6 +106,214 @@ tests:
...
+      - env: RELATED_IMAGE_OPERATOR
+        name: pipeline:cincinnati-operator
+      - env: GRAPH_DATA_IMAGE
+        name: pipeline:cincinnati-graph-data-container
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: hypershift-hostedcluster-workflow
```
Comment (wking) [inline ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml:247]: Moving from cluster-claims to HyperShift is interesting.  But before removing the WIP, we probably want to pick one approach and consolidate around that.

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**